config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: test
+
+production:
+  adapter: redis
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  channel_prefix: check_production
```

Comment:
We don't use websockets, do we need this file?

Reply:
this is one of the things that came out of reading `rake app:update`. I don't believe we need it explicitly (the Rails 6.0 branch that I had already merged passed without it), but we do have actioncable included as one of our rails dependencies in the gemfile.lock so I erred toward including it as an example in future, which is same approach I took with other autogenerated config, partly to reduce the diff we have to manage in future. I think I'd like to keep the file, but comment it out to reduce confusion. what do you think?

Reply:
Yes, add but comment sounds good to me! I prefer to comment than generating confusion in the future, specially that it references an environment variable.

Reply:
yeah that makes total sense! I'll update